go get fails: cli.StringFlag does not implement cli.Flag

When I tried to install using go get:
```
$ go get github.com/buildkite/terminal-to-html/cmd/terminal-to-html
# github.com/buildkite/terminal-to-html/cmd/terminal-to-html
gopath/src/github.com/buildkite/terminal-to-html/cmd/terminal-to-html/terminal-to-html.go:92:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in array or slice literal:
	cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
gopath/src/github.com/buildkite/terminal-to-html/cmd/terminal-to-html/terminal-to-html.go:97:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
	cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
gopath/src/github.com/buildkite/terminal-to-html/cmd/terminal-to-html/terminal-to-html.go:102:13: cannot use func literal (type func(*cli.Context)) as type cli.ActionFunc in assignment
```

```
$ go version
go version go1.10.4 linux/amd64
```

The same error for me:
```bash
$ go get github.com/buildkite/terminal-to-html/cmd/terminal-to-html
# github.com/buildkite/terminal-to-html/cmd/terminal-to-html
../../go/src/github.com/buildkite/terminal-to-html/cmd/terminal-to-html/terminal-to-html.go:92:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
	cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
../../go/src/github.com/buildkite/terminal-to-html/cmd/terminal-to-html/terminal-to-html.go:97:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in slice literal:
	cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
../../go/src/github.com/buildkite/terminal-to-html/cmd/terminal-to-html/terminal-to-html.go:102:13: cannot use func literal (type func(*cli.Context)) as type cli.ActionFunc in assignment
```

```bash
$ go version
go version go1.14 linux/amd64
```

I just installed go and am running into the same issues despite the fix:

```
$ go version
go version go1.14.2 linux/amd64
```
Simple install:
```
$ go get github.com/buildkite/terminal-to-html/cmd/terminal-to-html
# github.com/buildkite/terminal-to-html/cmd/terminal-to-html
../go/src/github.com/buildkite/terminal-to-html/cmd/terminal-to-html/terminal-to-html.go:92:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
../go/src/github.com/buildkite/terminal-to-html/cmd/terminal-to-html/terminal-to-html.go:97:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
../go/src/github.com/buildkite/terminal-to-html/cmd/terminal-to-html/terminal-to-html.go:102:13: cannot use func literal (type func(*cli.Context)) as type cli.ActionFunc in assignment
```

Other attempt:
```
$ GO111MODULE=on go get github.com/buildkite/terminal-to-html/cmd/terminal-to-html
go: downloading github.com/buildkite/terminal-to-html v1.0.2
go: downloading github.com/buildkite/terminal-to-html v3.2.0+incompatible
go: found github.com/buildkite/terminal-to-html/cmd/terminal-to-html in github.com/buildkite/terminal-to-html v3.2.0+incompatible
go: finding module for package github.com/codegangsta/cli
go: finding module for package github.com/buildkite/terminal
go: downloading github.com/buildkite/terminal v1.0.2
go: downloading github.com/codegangsta/cli v1.22.4
go: downloading github.com/buildkite/terminal v3.2.0+incompatible
go: found github.com/buildkite/terminal in github.com/buildkite/terminal v3.2.0+incompatible
go: found github.com/codegangsta/cli in github.com/codegangsta/cli v1.22.4
go: github.com/buildkite/terminal-to-html/cmd/terminal-to-html imports
        github.com/codegangsta/cli: github.com/codegangsta/cli@v1.22.4: parsing go.mod:
        module declares its path as: github.com/urfave/cli
                but was required as: github.com/codegangsta/cli
```

Am I missing something?

--- a/Readme.md
+++ b/Readme.md
@@ -49,7 +49,7 @@ You can use the provided `link.sh` to produce this escape sequence.
 If you have Go installed you can simply run the following command to install the `terminal-to-html` command into `$GOPATH/bin`:
 
 ```bash
-$ go get github.com/buildkite/terminal-to-html/cmd/terminal-to-html
+$ GO111MODULE=on go get github.com/buildkite/terminal-to-html/v3/cmd/terminal-to-html
 ```
 
 You can also just download the standalone binary from [https://github.com/buildkite/terminal-to-html/releases](https://github.com/buildkite/terminal-to-html/releases)

--- a/cmd/terminal-to-html/terminal-to-html.go
+++ b/cmd/terminal-to-html/terminal-to-html.go
@@ -9,7 +9,7 @@ import (
 	"net/http"
 	"os"
 
-	"github.com/buildkite/terminal-to-html"
+	"github.com/buildkite/terminal-to-html/v3"
 	"github.com/urfave/cli"
 )
 

--- a/go.mod
+++ b/go.mod
@@ -2,4 +2,4 @@ module github.com/buildkite/terminal-to-html
 
 go 1.12
 
-require github.com/urfave/cli v1.22.2
+require github.com/urfave/cli v1.22.4

--- a/go.mod
+++ b/go.mod
@@ -1,4 +1,4 @@
-module github.com/buildkite/terminal-to-html
+module github.com/buildkite/terminal-to-html/v3
 
 go 1.12
 

--- a/terminal.go
+++ b/terminal.go
@@ -6,7 +6,7 @@ https://raw.githubusercontent.com/buildkite/terminal-to-html/master/assets/termi
 and wrapped in a term-container div.
 
 You can call this library from the command line with terminal-to-html:
-go install github.com/buildkite/terminal-to-html/cmd/terminal-to-html
+GO111MODULE=on go install github.com/buildkite/terminal-to-html/v3/cmd/terminal-to-html
 */
 package terminal
 

